cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocProvisioner.java

```diff
@@ -429,8 +428,8 @@ private String getNormalizedProfileName(ProvisionerContext context) {
   String getImageVersion(ProvisionerContext context, DataprocConf conf) {
     String imageVersion = conf.getImageVersion();
     if (imageVersion == null) {
-      // Dataproc 2.0 is the default version until 6.9.x
-      imageVersion = "2.0";
+      // Dataproc 2.1 is the default version from 6.10.0 and later
+      imageVersion = "2.1";
```

Comment:
Why not to set this through configs rather than hard coding it?

Reply:
Do you mean we should set it through `CConf`?

Reply:
Generally, extension-specific properties are not put in `cdap-default.xml`.

Reply:
I was pointing to DataprocConf. But you are right, we can change it from clh. LGTM.